target/bin/listmailuser

```diff
@@ -23,12 +24,16 @@ while read -r LINE
 do
   USER=$(echo "${LINE}" | cut -d'|' -f1)
 
-  # ${QUOTA[0]} => current size
-  # ${QUOTA[1]} => configured size limit
-  # ${QUOTA[2]} => usage in percent
-  IFS=' ' read -r -a QUOTA <<< "$(doveadm quota get -u "${USER}" | tail +2 | awk '{ if ($3 == "STORAGE") { print $4" "$5" "$6 } }')"
-
-  echo "* ${USER} ( $(dovecot_quota_to_hr "${QUOTA[0]}") / $(dovecot_quota_to_hr "${QUOTA[1]}") ) [${QUOTA[2]}%]"
+  if [[ "${ENABLE_QUOTAS}" -eq 1 ]]
```

Comment:
```suggestion
  if [[ "${ENABLE_QUOTAS}" == 0 ]]
```

Would have worked instead of `-eq 0` btw. Your earlier finding with an expression failing because the value was not an integer to compare with `-eq` when ENV values are never really integers but strings that bash at some cases decides can qualify as integers if a number (with the unset case being coerced to NaN presumably).

I think it's good though with this change to check if it's enabled rather than disabled :+1:

Reply:
> == 0 would have worked instead of -eq 0 btw?

Yes. But as we work with "integers" 0/1, a string comparison feels wrong.
I like the -eq 1 approach from a logical point of view. Only if ENABLE_QUOTA has the value of 1 and is therefore set, query the dovecot quota. In all other cases, hide the quota information.